File: SoftUni/Exercises/Fundamentals/basicSyntaxConditionalStatementsLoops/PrintAndSum.java
package basicSyntaxConditionalStatementsLoops;

import java.util.Scanner;

public class PrintAndSum {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int start = sc.nextInt();
		int end = sc.nextInt();
		int sum = 0;
		for(int i = start; i <= end; i++) {
			System.out.print(i+" ");
			sum+=i;
		}
		System.out.println();
		System.out.print("Sum: " + sum);
		sc.close();
	}

}

File: SoftUni/Exercises/Advanced/JS Front-End - февруари 2023/fundamental syntax/printAndSum.js
function printAndSum(start, end) {
    let sum = 0;
    let nums = [];
    for(let i = start; i <= end; i++ ) {
        sum+=i;
        nums.push(i);
    }
    console.log(nums.join(" "));
    console.log('Sum: ' + sum);
}